android/app/src/main/java/org/fossasia/openevent/core/speaker/SpeakerDetailsActivity.java

```diff
@@ -286,6 +288,22 @@ private void hideEmptyURLButtons() {
         }
 
     }
+
+    private void loadData() {
+        speakerDetailsViewModel.getSpeaker(speakerName).observe(this, speakerData -> {
+            selectedSpeaker = speakerData;
+            loadSpeakerDetails();
+        });
+    }
+
+    //TODO: Workaround. Needs fix during implementation of RealmLiveData
+    @Subscribe
+    public void onBookmarksChanged(BookmarkChangedEvent bookmarkChangedEvent) {
+        Timber.d("Bookmarks Changed");
+        loadData();
+    }
```

Comment:
This needs to be added in onResume

And the solution should not contain event handling

Reply:
> This needs to be added in onResume
You mean the TODO?

> And the solution should not contain event handling
How do you suggest I do this @iamareebjamal ? I'm relatively new to the project and I came to the solution after looking at how things were done in TrackSessionsActivity.java. setBookmark() in RealmDataRepository class posts the event so my first thought was to add this method onBookmarksChanged(). If there are other ways to do this, suggest me and I'll try that. Thanks!

Reply:
> You mean the TODO?
Yes

And about the event based method, yes this is how we are doing things currently in the project but we are refactoring code to avoid this and hence we don't want to add new code using this bad practice. The LiveData itself should handle the reactivity and there should be no need of the change event

Reply:
Okay, thanks for the clarification @iamareebjamal! For now, just moved the TODO.

Reply:
The event bus pattern is still used

Reply:
> We can accept these changes. But write a TODO, that this is a workaround and fix needs to be done in the implementation of RealmLiveData

Isn't this why we added a TODO @iamareebjamal? I understand this isn't what you want in future but your earlier instruction was to include a TODO so I did likewise. If a solution with RealmLiveData is expected at the moment then I'll have to learn about LiveData more before I come up with the solution.